src/doc/book/error-handling.md

```diff
@@ -225,7 +225,7 @@ sense to put it into a function:
 ```rust
 # fn find(_: &str, _: char) -> Option<usize> { None }
 // Returns the extension of the given file name, where the extension is defined
-// as all characters proceeding the first `.`.
+// as all characters following the first `.`.
```

Comment:
>  I believe the proceeding/following distinction is a bug (but might be a English dialect distinction?).

precede/proceed are two different things, maybe that's what you're getting at? proceed and follow are synonyms. That said, "follow" might be more clear, so I'm fine with changing it.